Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.7-slim
+
+LABEL Name=morandirb Version=0.0.1
+
+RUN apt-get update && apt-get install -yyq --no-install-recommends \
+  build-essential \
+  libglib2.0-dev \
+  libgtk2.0-dev \
+  libgdk-pixbuf2.0-dev \
+  libtiff5-dev \
+  imagemagick \
+  liblcms2-utils \
+  && apt-get clean \
+  && rm -rf /va/lib/apt/lists/*
+
+COPY Gemfile* /app/
+WORKDIR /app/
+RUN bundle config --local gemfile Gemfile.docker
```

Comment:
Makes sense; `Gemfile.docker` for devs and `Gemfile`/`gemspec` for public consumption.

Reply:
I would be nicer to only have one Gemfile, I'm sure, but unfortunately I could not make it play nicely with docker ...